src/game/chars/CCharSkill.cpp

```diff
@@ -3559,9 +3559,6 @@ int CChar::Skill_Stage( SKTRIG_TYPE stage )
 {
 	ADDTOCALLSTACK("CChar::Skill_Stage");
     SKILL_TYPE skill = Skill_GetActive();
-    if (skill == SKILL_NONE)
```

Comment:
Is there a particular reason to remove it? Does it cause some bug? I have a great fear changing the skill system lol

Reply:
It wasn't there at all, I added it on my pr while recode Fishing system, but it seems like Focus skill cause a bug when I add this like, and it keeps cancelling targets in every 2 seconds, so I cancelled this like to make sure it works like before.